src/RimDev.Stuntman.AspNetCore/common/StuntmanOptions.cs

```diff
@@ -83,6 +83,16 @@ public StuntmanOptions AddUser(StuntmanUser user)
             return AddUser(user, Constants.StuntmanOptions.LocalSource);
         }
 
+        public StuntmanOptions AddUsers(IEnumerable<StuntmanUser> users) 
```

Comment:
Would you mind adding `ThrowsForNullUser` and `AddsUser` tests for this method in [tests/Core.AspNetCore.Tests/common/StuntmanOptionsTests.cs](https://github.com/ritterim/stuntman/blob/master/tests/Core.AspNetCore.Tests/common/StuntmanOptionsTests.cs)?